tool/main.re

```diff
@@ -250,7 +245,32 @@ static MULTIPLAYER_MENU = Ui::new("Multiplayer:", List::of(
         onclick: fn(label: Text) { leave_ui(); },
     }),
 ));
+
 static UTIL_MENU = Ui::new("Util:", List::of(
+    UiElement::Input(Input {
+        label: Text { text: "Save Recording" },
+        input: "",
+        onclick: fn(input: string) {
+            Tas::save_recording(input, TAS_STATE.recording);
```

Comment:
Can you move this into a function within `tas.re` so that `main.re` doesn't need to know about the internals of `tas.re` like the existence of the `TAS_STATE` static?